config/environment.rb

```diff
@@ -3,8 +3,10 @@
 # Load the Rails application.
 require_relative "application"
 
-OpenTelemetry::SDK.configure do |c|
-  c.use_all
+if ENV["OTEL_EXPORTER_OTLP_ENDPOINT"].present? && !Rails.const_defined?(:Console)
```

Comment:
Oh, so don't run it in the console, ever? that's what that `!Rails.const_defined?(:Console)` does?
neat!